lib/active_job/queue_adapters/shoryuken_concurrent_send_adapter.rb

```
@@ -6,33 +6,50 @@
-    #
-    # config.active_job.queue_adapter = adapter
+    #   config.active_job.queue_adapter = adapter
     class ShoryukenConcurrentSendAdapter < ShoryukenAdapter
+      # Initializes a new concurrent send adapter
+      #
+      # @param success_handler [Proc, nil] callback for successful enqueues
+      # @param error_handler [Proc, nil] callback for failed enqueues
       def initialize(success_handler = nil, error_handler = nil)
         @success_handler = success_handler
         @error_handler = error_handler
       end
```

RuboCop (1.81.7)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)